sql/hive/src/main/scala/org/apache/spark/sql/hive/hiveUdfs.scala

```diff
@@ -321,6 +324,18 @@ private[hive] case class HiveGenericUdtf(
   override def toString = s"$nodeName#${funcWrapper.functionClassName}(${children.mkString(",")})"
 }
 
+/**
+ * Resolve Udtfs Alias.
+ */
+private[spark] object ResolveUdtfsAlias extends Rule[LogicalPlan] {
+  def apply(plan: LogicalPlan) = plan transform {
+    case q: LogicalPlan => q transformExpressions {
+      case MultiAlias(udtf@HiveGenericUdtf(_, _, _), names) =>
```

Comment:
A better idea is to convert the `MultiAlias(udtf@HiveGenericUdtf(_, _, _), names)` => `HiveGenericUdtf(_, names, _)`? And the `MultiAlias` just like the `UnresolvedAttribute`, which will be eliminated after `Analyzing`